### ISSUE TITLE
support: Action failed with "not found deploy key or tokens"

### ISSUE THREAD

**Issue by williamd5** [peaceiris/actions-gh-pages#670]
### Checklist

- [X] I am using the latest version of this action.
- [X] I have read the latest README and followed the instructions.
- [X] I have read the latest GitHub Actions official documentation and learned the basic spec and concepts.

### Describe your question

The deploy job returns an error and the workflow fails

### Relevant links

```markdown
Public repository: https://github.com/williamd5/AlgorithmBibliotheca
YAML workflow: https://github.com/williamd5/AlgorithmBibliotheca/blob/a2e4d1fedc11d33203cb1db706d212199de62876/.github/workflows/jsdoc.yml
```

### Relevant log output

```
Run peaceiris/actions-gh-pages@v3
[INFO] Usage https://github.com/peaceiris/actions-gh-pages#readme
Dump inputs
Setup auth token
  Error: Action failed with "not found deploy key or tokens"
```

### Additional context.

I am using PAT with the following scopes enabled:
![image](https://user-images.githubusercontent.com/66572326/141980613-bc76238a-b9f5-4f95-89aa-de6013124747.png)

**Comment by peaceiris** [comment]
Why don't you use the recommended snippet on README?

```yaml
- name: Deploy
  uses: peaceiris/actions-gh-pages@v3
  with:
    github_token: ${{ secrets.GITHUB_TOKEN }}
    publish_dir: ./public
```

> For newbies of GitHub Actions: Note that the GITHUB_TOKEN is NOT a personal access token. A GitHub Actions runner automatically creates a GITHUB_TOKEN secret to authenticate in your workflow. So, you can start to deploy immediately without any configuration.

**Comment by williamd5** [comment]
I copied the example from the GitHub marketplace readme. The repo readme suggests multiple options for tokens/keys without much explanation and the pages example from the marketplace (with `ACTIONS_DEPLOY_KEY`) made sense on a deploy job. I did note that the secret is created automatically earlier in the workflow and I have not set my PAT in the repo secrets.

Thanks for your help, using `github_token: ${{ secrets.GITHUB_TOKEN }}` resolved this issue.
